core/tests/puppeteer-acceptance-tests/user-utilities/super-admin-utils.js

```diff
@@ -101,4 +101,444 @@ module.exports = class e2eSuperAdmin extends baseUser {
     showMessage(`User ${username} does not have the ${role} role!`);
     await this.goto(currentPageUrl);
   }
+
+  async editClassroom({ topicId }) {
+    await this.goto('http://localhost:8181/admin#/config');
```

Comment:
This is actually wrong, I think ... classroom editing happens at /classroom-admin. We are deprecating the thing in /admin. 

/cc @Nik-09 @lkbhitesh07 -- we should get your PRs in soon, it looks like this is causing confusion.

Reply:
Absolutely @seanlip, I think there is only one e2e test failing on @Nik-09's PR, We will get the deprecation done by the weekend.
Thanks

Reply:
The feature related to this review thread was removed.